src/renate/updaters/experimental/offline_er.py

```diff
@@ -66,17 +65,32 @@ def on_model_update_start(
         self._num_points_current_task = len(train_dataset)
 
     def train_dataloader(self) -> DataLoader:
-        train_loader = super().train_dataloader()
-        loaders = {"current_task": train_loader}
-        if len(self._memory_buffer) > self._memory_batch_size:
+        """Returns the dataloader for training the model."""
+        loaders = {}
+        memory_batch_size = int(self._batch_memory_frac * self._batch_size)
+        if len(self._memory_buffer) >= memory_batch_size:
```

Comment:
Not directly related to this PR, but would it make sense to add a check somewhere that the buffer size must always be larger than the memory size? Otherwise the memory is always ignored.